openedx/core/djangoapps/course_date_signals/handlers.py

```diff
@@ -91,13 +91,25 @@ def _get_custom_pacing_children(subsection, num_weeks):
     Return relative date items for the subsection and its children
     """
     items = [subsection]
+    all_problems_are_ora = True
+    has_empty_unit = True
     section_date_items = []
     while items:
         next_item = items.pop()
+        if next_item.category != 'sequential' and next_item.category != 'vertical':
```

Comment:
I'd probably write this as `is_hierarchy = next.item.category in ['sequential', 'vertical']` and then use that variable in both places you check.